our_land/settings.py

```diff
@@ -0,0 +1,175 @@
+import os
+from collections import OrderedDict
+
+import dj_database_url
+
+from hurumap.settings import *  # noqa
+BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+
+INSTALLED_APPS = ['our_land'] + INSTALLED_APPS
+
+STATIC_ROOT = os.path.join(BASE_DIR, 'static')
+
+DATABASE_URL = os.environ.get('DATABASE_URL',
+                              'postgresql://our_land:our_land@localhost/our_land')
+DATABASES['default'] = dj_database_url.parse(DATABASE_URL)
+DATABASES['default']['ATOMIC_REQUESTS'] = True
+
+# Localise this instance of HURUmap
```

Comment:
Do
```python
OUR_LAND = HURUMAP

OUR_LAND['name'] = 'Our Land'

...

```
This means all settings will be set on `OUR_LAND` and no need to remember which settings goes to `HURUMAP`, etc.